langfuse/callback.py

```diff
@@ -34,14 +35,17 @@ def __init__(
         statefulTraceClient: Optional[StatefulTraceClient] = None,
         release: Optional[str] = None,
     ) -> None:
+        public_key = public_key if public_key else os.environ.get("LF_PK")
+        secret_key = secret_key if secret_key else os.environ.get("LF_SK")
+
         # If we're provided a stateful trace client directly
         if statefulTraceClient:
             self.trace = Run(statefulTraceClient, None)
             self.runs = {}
 
         # Otherwise, initialize stateless using the provided keys
         elif public_key and secret_key:
-            self.langfuse = Langfuse(public_key, secret_key, host, debug=debug, release=release)
+            self.langfuse = Langfuse(debug=debug, release=release)
```

Comment:
Should we set the keys here?

Reply:
Yes makes more sense above we are accessing it so here we can keep it